src/clearskies/columns/has_many.py

```diff
@@ -463,12 +463,32 @@ def __get__(self, instance: Model, cls: type[Model]) -> Model:
     def __get__(self, model, cls):
         if model is None:
             self.model_class = cls
-            return self  # type:  ignore
+            return self  # type: ignore
 
         # this makes sure we're initialized
         if "name" not in self._config:  # type: ignore
             model.get_columns()
 
+        # The data will be in model._data[column_name] as a list of child models
+        raw_data = model.get_raw_data()
+
+        if self.name in raw_data and isinstance(raw_data[self.name], list):
```

Comment:
The cache needs to go in `model._transformed_data`.  The raw data is only supposed to be exactly what came out of the database.